### PR TITLE
[terra-hyperlink] Fix focus indicator

### PR DESCRIPTION
**Pull request by cm9361**
### Summary

**What was changed:**
The focus indicators for the default and lowlight theme were fixed for the Terra Hyperlink component.

**Why it was changed:**
The change was made to give consistency within the theme for the Hyperlink and Terra components that consume the control.

### Testing

This change was tested using:

- [X] WDIO
- [ ] Jest
- [ ] Visual testing (please attach a screenshot or recording)
- [ ] Other (please describe below)
- [ ] No tests are needed

### Reviews

In addition to engineering reviews, this PR needs:

- [X] UX review
- [ ] Accessibility review
- [ ] Functional review

### Additional Details

**This PR resolves:**

UXPLATFORM-10326 

---

Thank you for contributing to Terra.
@cerner/terra

### CODE REVIEW

File: packages/terra-hyperlink/tests/wdio/hyperlink-spec.js
Comment (sycombs) [inline packages/terra-hyperlink/tests/wdio/hyperlink-spec.js:1]: Just checking, are the tests that were removed covered by the other existing tests?

Comment (cm9361) [reply packages/terra-hyperlink/tests/wdio/hyperlink-spec.js:1]: @sycombs - The tests that were removed were hard-coding the styles and not reflecting the true nature of the control.  We do have WDIO tests to cover the focus indicators.

**Review by sycombs — APPROVED** [review]

**Review by adoroshk — APPROVED** [review]
Works as expected. LGTM
Screenshot 2024-03-29 at 9 46 45 AM

**Comment by mjpalazzo** [comment]
@cm9361 - the visual focus indicator on clinical lowlight and terra default are working as required.

**Comment by mjpalazzo** [comment]
@cm9361 - you have a UX review label. Do you want @eawww to approve as well?

**Comment by cm9361** [comment]
> @cm9361 - the visual focus indicator on clinical lowlight and terra default are working as required.

@mjpalazzo  - I think your approval is good enough in my opinion.

**Pull request merged**